src/python/grpcio_health_checking/grpc/health/v1/health.py

```diff
@@ -44,10 +46,7 @@ def __init__(self):
   def Check(self, request, context):
     with self._server_status_lock:
       if request.service not in self._server_status:
```

Comment:
Eliminate the double-dipping into the dictionary:
```
status = self._server_status.get(request.service)
if status is None:
  context.set_code(grpc.StatusCode.NOT_FOUND)
  return health_pb2.HealthCheckResponse()
else:
  return health_pb2.HeathCheckResponse(status=status)
```
.

_Technically_ we don't support fail-to-return-a-value-from-a-response-unary-method-implementation-but-it's-all-cool-because-we-set-a-non-OK-code. It happens to behave compatibly today but let's not put it into this code until after we've decided to certify it as a proper feature and added test coverage?